src/pathwayvis/components/reactionpanel/changeBounds/changeBoundsPanel/changeBoundsPanel.component.html

```diff
@@ -0,0 +1,55 @@
+<!--Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.-->
+
+<!--Licensed under the Apache License, Version 2.0 (the "License");-->
+<!--you may not use this file except in compliance with the License.-->
+<!--You may obtain a copy of the License at-->
+
+<!--http://www.apache.org/licenses/LICENSE-2.0-->
+
+<!--Unless required by applicable law or agreed to in writing, software-->
+<!--distributed under the License is distributed on an "AS IS" BASIS,-->
+<!--WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.-->
+<!--See the License for the specific language governing permissions and-->
+<!--limitations under the License.-->
+
+<div ng-repeat="item in $ctrl.items">
+    <br>
+    <md-list-item ng-click="$ctrl.clickedItemFunction({item: item})">
+      <span style="float: left; font-size: 14px" class="md-caption">{{ item.lower_bound }}</span>
+      <span style="margin-left: 18px; margin-right: 18px;">
+        <= <span class="md-caption">{{ $ctrl.changedReactionDisplay(item) }}</span>
+      <= </span>
+        <span style="float: right; font-size: 14px" class="md-caption">{{ item.upper_bound }}</span>
+        <md-button class="md-icon-button md-secondary" ng-click="$ctrl.onResetBounds({item: item})" style="float: right">
+          <md-icon md-font-icon="close">close</md-icon>
+        </md-button>
+    </md-list-item>
+    <div style="border:1px solid black; height: 130px; padding: 2%; margin-top: 5%; text-align: center" ng-show="$ctrl.clickedItem == item.id || ($first && !$ctrl.clickedItem) || $ctrl.items.length == 1">
+      <md-list-item ng-if="item">
+        <input style="float: left; width: 53px" type="number" ng-init="$ctrl.lowerbound=item.lower_bound"  ng-model="$ctrl.lowerbound"/>
```

Comment:
Please don't use `ng-init`, it's an anti-pattern: https://docs.angularjs.org/api/ng/directive/ngInit do this in the constructor / $onInit life cycle hook